prioritizer/prioritizer.py

```diff
@@ -5,106 +5,95 @@
 import re
 import json
 import pandas as pd
+from pydantic import BaseModel
+from typing import List
+from pathlib import Path
+from utils.reading_data import read_city_inventory, read_actions
+from utils.additional_scoring_functions import count_matching_hazards, find_highest_emission
+from utils.prompt import prompt as PROMPT
 
 load_dotenv()
 
 api_key = os.getenv("OPENAI_API_KEY")
 client = OpenAI(api_key=api_key)
 
 # Constants for quantitative scoring
-SCORE_MAX = 100 / 6
-MAX_EMISSIONS_REDUCTIONS = 500000
+SCORE_MAX = 100 / 4
+# Do a dynamic adaptation of how many fields we calculate for an action ( if there are nulls )
 scale_scores = {
     "Very High": 1.0,
     "High": 0.75,
     "Medium": 0.5,
     "Low": 0.25,
     "Very Low": 0.0,
 }
-MAX_TIME_IN_YEARS = 20
-MAX_COST = 60000000
+scale_adaptation_effectiveness ={
+    'low': 0.33,
+    'medium': 0.66,
+    'high': 0.99
+}
 
-# File paths
-OUTPUT_FILE = "new_output.json"
+# Define the mapping for timeline options
+timeline_mapping = {
+    "<5 years": 1.0,
+    "5-10 years": 0.5,
+    ">10 years": 0
+}
+ghgi_potential_mapping = {
+    "0-19" : 0.20,
+    "20-39" : 0.4,
+    "40-59" : 0.6,
+    "60-79" : 0.8,
+    "80-100" : 1.0
+}
+# Dynamically construct the file paths based on the script's location
+BASE_DIR = Path(__file__).resolve().parent
```

Comment:
This is not the data from the main branch right?
We should change this to actually using the main branch data so make sure it works with that data